alvr/client_openxr/src/stream.rs

```diff
@@ -200,45 +194,35 @@ impl StreamContext {
             interaction_ctx.read().hands_interaction[1].controllers_profile_id,
         );
 
-        let input_thread_running = Arc::new(RelaxedAtomic::new(true));
+        let input_thread_running = Arc::new(RelaxedAtomic::new(false));
 
-        let reference_space = Arc::new(interaction::get_reference_space(
+        let stage_reference_space = Arc::new(interaction::get_reference_space(
             &xr_session,
             xr::ReferenceSpaceType::STAGE,
         ));
+        let view_reference_space = Arc::new(interaction::get_reference_space(
+            &xr_session,
+            xr::ReferenceSpaceType::VIEW,
+        ));
```

Comment:
Why do this when you're just gonna update it immediately after anyway?

Reply:
Because the fields are not Option. It would be more hassle making it Option, because then we need unwrap or "?"

Reply:
so what I assumed. sure